deploy/charts/cert-manager/values.yaml

```diff
@@ -88,6 +88,10 @@ ingressShim: {}
   # defaultACMEChallengeType: ""
   # defaultACMEDNS01ChallengeProvider: ""
 
+# Enable Prometheus scrapping
+metrics:
+  enabled: true
```

Comment:
I'm tempted to not bother exposing this in values.yaml. Setting `enabled=false` won't actually disable metrics, and could mislead users into thinking it does.

Perhaps lets just add the annotations and omit the values.yaml changes, and if there's a request in future to *not* collect metrics altogether we can then add this option 😄